pkg/constants/constants.go

```diff
@@ -133,7 +133,9 @@ const (
 	AnsibleSSHInventoryParams                    = "-o StrictHostKeyChecking=no"
 	AnsibleExtraVarsFlag                         = "--extra-vars"
 	CloudCLIBranch                               = "custom-vms-cloud-10"
-	DefaultConfigFileName                        = ".avalanche-cli"
+	MetricsEnabled                               = "MetricsEnabled"
```

Comment:
add constants for old default configs paths, and new default config path, absolute ones